### PR TITLE
Omnia/feature/#77 tax interface

### LINKED ISSUE #61: Implement Tax Interface proposal

### PR DESCRIPTION
**Pull request by devjk1**
closes #61 

Support package
-added TaxRequest interface
-added TaxRequestItem interface
-extended TipoffPackage with hasBindings helper
-added bindings to TipoffServiceProvider

### CODE REVIEW

**Comment by drewroberts** [comment]
@pdbreen Can you take a look at this?

File: src/TipoffPackage.php [hunk 1]
```diff
@@ -96,4 +96,11 @@ public function hasNovaResources(array $novaResources): self
 
         return $this;
     }
+
+    public function hasBindings(array $bindings): self
+    {
+        $this->bindings = array_merge($this->bindings, $bindings);
```
Comment (pdbreen) [inline src/TipoffPackage.php:104]: Like other types, `bindings` should be declared as a class property, initialized to empty array.

**Review by pdbreen — COMMENTED** [review]
Since there is existing test coverage for TipoffPackage/TipoffServiceProvider, its best practice to extend the tests for the new functionality.

Also, one specific change needed.

**Review by pdbreen — APPROVED** [review]
looks ok to me

**Review by drewroberts — APPROVED** [review]
Thank you! 🌜

**Pull request merged**